test/configCases/output/inner-dirs-entries/dep.js

```diff
@@ -0,0 +1,6 @@
+import glob from 'glob';
+
+export default function someDummyFunction() {
+    typeof glob;
```

Comment:
This is a very fragile test. Use something like `expect(typeof glob).toBe("object")`